extensions/KHR/SPV_KHR_quad_control.asciidoc

```diff
@@ -0,0 +1,183 @@
+SPV_KHR_quad_control
+==================
+
+Name Strings
+------------
+
+SPV_KHR_quad_control
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Tobias Hector, AMD
+- Nicolai Haehnle, AMD
+- Jeff Bolz, Nvidia
+- Graeme Leese, Broadcom
+
+Notice
+------
+
+Copyright (c) 2024 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Draft
```

Comment:
So sorry I didn't notice this previously.  I think the "status" needs to be updated.  The minimum change would be:

```suggestion
- Complete
```

Or, since this is a KHR extension, we could say:

```
- Approved by the SPIR-V Working Group: 2023-12-06
- Approved by the Khronos Board of Promoters: 2024-01-19
```

Reply:
Done and rebuilt.